policy-test/tests/e2e_failure_accrual.rs

```diff
@@ -0,0 +1,63 @@
+use linkerd_policy_test::{
+    annotate_service, bb, create, create_ready_pod, curl, with_temp_ns, LinkerdInject,
+};
+
+#[tokio::test(flavor = "current_thread")]
+async fn consecutive_failures() {
+    const MAX_FAILS: usize = 5;
+    const REQUESTS: usize = 20;
+    with_temp_ns(|client, ns| async move {
+        // Create a bb service with two pods, one of which always returns 200
+        // OK, and the other of which always returns 500 Internal Server Error.
+        let good_pod = bb::Terminus::new(&ns)
```

Comment:
Is there any purpose to including a good pod in the service?  It seems like it makes the test less deterministic since we don't know what proportion of requests will actually go to each pod.  There is even a chance that the circuit breaker may not trip during this test if too many requests go to the good pod.

If we took this out and tested with only the bad pod in the Service, we could even assert that the number of failures is exactly 5, which seems nice.

Reply:
Hmm, that's actually a good point.. I hadn't considered that we could just run this test with only a failing endpoint. Although every other request would fail with a 503, since the load balancer would have no endpoints in it and the requests would time out.